wallet/src/libwallet/api.rs

```diff
@@ -264,12 +264,7 @@ where
 	/// sender as well as the private file generate on the first send step.
 	/// Builds the complete transaction and sends it to a grin node for
 	/// propagation.
-	pub fn file_finalize_tx(&mut self, receiver_file: &str) -> Result<Slate, Error> {
-		let mut pub_tx_f = File::open(receiver_file)?;
-		let mut content = String::new();
-		pub_tx_f.read_to_string(&mut content)?;
-		let mut slate: Slate = json::from_str(&content).map_err(|_| ErrorKind::Format)?;
-
+	pub fn file_finalize_tx(&mut self, mut slate: Slate) -> Result<Slate, Error> {
```

Comment:
Maybe make the `file_receive_tx` method consistent with this change then?

Reply:
Sure I will actually merge the two PRs and add this.